source/LocationFilter.cpp

```diff
@@ -274,6 +276,10 @@ void LocationFilter::Save(DataWriter &out) const
 		}
 		if(center)
 			out.Write("near", center->TrueName(), centerMinDistance, centerMaxDistance);
+		if(planetIsVisited)
+			out.Write("visited", "planet");
+		else if(systemIsVisited)
```

Comment:
Why else? Cannot both be used at the same time?

Reply:
If you've visited a planet, then you've certainly visited the system that the planet is in. If a location filter has `visited planet` but it's actually used to filter systems, then it'll check if the system has been visited.

Reply:
Okay, so the syntax simply doesn't support both for now.